integrations/tensorflow-keras/setup.py

```diff
@@ -0,0 +1,37 @@
+from setuptools import setup
+
+
+def main():
+    with open('README.md') as readme_file:
+        readme = readme_file.read()
+
+    extras = {
+    }
+
+    all_deps = []
+    for group_name in extras:
+        all_deps += extras[group_name]
+    extras['all'] = all_deps
+
+    base_libs = ['neptune-client>=0.4.126']
+
+    setup(
+        name='neptune-tensorflow-keras',
+        version="0.0.1", # TODO fix version generation, propably use versioneer
```

Comment:
Can we release different pip packages in this repo separately? Should we? How would it work?

Reply:
Lets make this a separe MR - I suggest using different tag regexps - for example `tensorflow-keras/a.b.c` would release only this integration while `a.b.c` would release main package

Reply:
Ok.